packages/vscode-extension/src/extension.ts

```diff
@@ -11,16 +11,19 @@ let client: LanguageClient;
 
 export function activate(context: ExtensionContext) {
   // The server is implemented in node
-  const serverModule = context.asAbsolutePath(
-    path.join('..', 'language-server', 'dist', 'cypher-language-server.js'),
+  const runServer = context.asAbsolutePath(
+    path.join('dist', 'cypher-language-server.js'),
+  );
+  const debugServer = context.asAbsolutePath(
+    path.join('..', 'language-server', 'dist', 'server.js'),
```

Comment:
Use a different configuration for debug and for running (once the extension is compiled, the language server has to be living inside it, we cannot use a relative path). Also we want that language server to be minified.